recipes/libcurl/all/conanfile.py

```diff
@@ -320,6 +324,12 @@ def _get_configure_command_args(self):
 
         if self._has_metalink_option:
             params.append("--with-libmetalink={}".format(yes_no(self.options.with_libmetalink)))
+        
+        if not self.options.with_proxy:
+            params.append("--disable-proxy")
+       
+        if not self.options.with_rtsp:
+            params.append("--disable-rtsp")
```

Comment:
it would be better to be explicit and this in initialization of params:

```python
        params = [
            "--with-libidn2={}".format(yes_no(self.options.with_libidn)),
            "--with-librtmp={}".format(yes_no(self.options.with_librtmp)),
            "--with-libpsl={}".format(yes_no(self.options.with_libpsl)),
            "--with-schannel={}".format(yes_no(self.options.with_ssl == "schannel")),
            "--with-secure-transport={}".format(yes_no(self.options.with_ssl == "darwinssl")),
            "--with-brotli={}".format(yes_no(self.options.with_brotli)),
            "--enable-shared={}".format(yes_no(self.options.shared)),
            "--enable-static={}".format(yes_no(not self.options.shared)),
            "--enable-ldap={}".format(yes_no(self.options.with_ldap)),
            "--enable-debug={}".format(yes_no(self.settings.build_type == "Debug")),
            "--enable-ares={}".format(yes_no(self.options.with_c_ares)),
            "--enable-threaded-resolver={}".format(yes_no(self.options.with_c_ares)),
            "--enable-proxy={}".format(yes_no(self.options.with_proxy)),
            "--enable-rtsp={}".format(yes_no(self.options.with_rtsp)),
        ]
```

Reply:
I did that initially but it fails to build with autoconf as those options don't take parameters so fails to build